core/templates/pages/volunteer-page/volunteer-page.component.html

```diff
@@ -1,56 +1,93 @@
 <div class="volunteer-page e2e-test-volunteer-page">
-  <div class="row oppia-volunteer-banner">
-    <div class="oppia-volunteer-banner-content flex-gt-md-55">
+  <div class="row oppia-volunteer-banner p-5">
+    <div class="oppia-volunteer-banner-content flex-gt-md-45">
       <h1 class="oppia-volunteer-h1">
-        Join our global team of volunteers
+        Join Us
```

Comment:
Can we internationalize these strings? See the https://github.com/oppia/oppia/wiki/How-to-develop-for-i18n and the codebase for examples.